### ISSUE TITLE
Add the Coffee module to improve the administrator experience

### ISSUE THREAD

**Issue by michaelmol** [acquia/reservoir#15]

**Comment by wimleers** [comment]
I see the appeal of https://www.drupal.org/project/coffee for Drupal's default install profiles: because there are SO many things you can do, and they're hidden behind so many layers.

However, in Reservoir, I don't think this is necessary. The four high-level tabs (Content/API/Content models/Access control) get you to within 1-2 clicks of >95% of what you need.

**Comment by tedbow** [comment]
I agree with @wimleers that I don't think it necessary. Like Wim said you can get to 95% of what you need by from top level menus. I think we should focus on keeping it that way.

Reservoir is opinionated and we are trying to surface certain and hide others.  Coffee search I think would give equal weight to items that are available in the "Manage the backend" section. Which is not the intention.

**Comment by wimleers** [comment]
@michaelmol Thoughts?
